src/content/docs/ai-observability/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Introduction to AI Monitoring'
+metaDescription: 'AI Monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import apmAiMonitoringAllCapabilities from 'images/apm_screenshot-full_ai-monitoring-all-capabilities.gif'
+
+Your AI-powered app introduces new technologies into your environment that are hard to monitor, let alone understand. With AI Monitoring, you can bridge unfamiliar technologies to familiar solutions, deepening your knowledge about your AI's behavior so you can improve its performance. 
+
+<img
+    title="View your app's AI data"
+    alt="A gif that shows where to find your AI data in New Relic"
+    src={apmAiMonitoringAllCapabilities}
+/>
+
+<figcaption>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI Monitoring. Choose your entity from the AI entities page to view your data.
+</figcaption>
+
+* **Evaluate errors and bugs on the code-level.** Get total insight over how your AI processes perform and make improvements when they fail. 
+* **View feedback from your end user.** AI Monitoring correlates prompts, AI responses, and user feedback with transaction ids so you can see where your AI assistant breaks down. 
+* **Track token usage.** See when token usage spike, then optimize your AI-powered app to keep costs down. 
+
+## Get started with AI Monitoring [#start-aim]
+
+AI Monitoring exposes your AI data as a subset of APM data. To get started, you need to:
+
+* [Create a New Relic account](https://newrelic.com/signup)
+* [Install one of our supported APM agents](/install/aim)
```

Comment:
Really, it's: Install the APM agent that's relevant to your AI app, right?

Reply:
Also, because it's only some of our APM agents, perhaps it's worth moving that table up from below.

Reply:
So it's more like: 

Get started with AI Monitoring

1. Make we have an agent that meets your needs:
[chart]
2. Go install it

I'm looking at the Install AI Monitoring doc and I see that it looks like only Node and Python are currently supported? 

Maybe it's worth considering combining this install doc with the intro doc?

Regardless, I think it's super important to clearly lay out what's available and what isn't in that intro doc.